assets/marketplace/main.js

```diff
@@ -0,0 +1,99 @@
+let vscode;
+try {
+    vscode = acquireVsCodeApi();
+} catch (error) {
+    
+}
+
+const app = new Vue({
+    el: '#app',
+    data: {
+        message: 'Hello Vue!',
+        searchInput: '',
+        selectedModule: {},
+        selectedTag: "",
+        moduleName: '',
+        modules: [],
+        endpoint: document.getElementById('app').getAttribute('data-endpoint'),
+        errorMessage: "",
+    },
+    created: async function () {
+        this.modules = await this.getModules();

```

Comment:
Do we have specific error page? What will be shown when there is error in api call?

Reply:
Added an error page